docs/user_guide.md

```diff
@@ -39,108 +41,145 @@ Sample usage:
 
 ### Chart Panel
-The chart panel contains two types of charts: ramp charts and total contribution bars. The length of the red bars is proportional to the total contribution of the author. If the author contributes ‘too much’ compared to other authors in the report, there will be multiple red bars for him. If the user hovers on the bar, the exact amount of contribution will be shown.
+The `Chart Panel` contains two type of indicators:
+- [Ramp Chart](#ramp-chart)
+- [Total Contribution Bar](#total-contribution-bars)
+
 #### Ramp Chart
-To illustrate frequency and amount of contribution in the same graph, and also allow easy comparison between we created a new type of visualization. We call it Ramp Charts. Here are some examples of the Ramp graphs:
+To illustrate frequency and amount of contribution in the same graph, and to allow easy comparison between each entry, we implemented a new type of visualization.
+This is referred as the `Ramp Charts`.
+
+Below are a few examples:
 
 ![Ramp Charts](images/rampchart.png)
-Each light blue bar represents the contribution timeline of an individual author for a specific repository. On each row, there are several ‘ramps’.
+Each light blue bar represents the contribution timeline of an individual author for a specific repository. Each row, there are several **ramps**.
 - The area of the ramp is proportional to the amount of contribution the author did at that time period.
 - The position of the right edge of the ramp (perpendicular to the blue bar) is dependent on the time period that ramp represents.
 - To make comparison between two authors easier, the color of the ramps that represent different authors' contributions at the same time period are the same.
 - The timelines (blue bar) of the charts should be aligned, so that the comparison of contribution between two authors are easier.
 - There is no limit to the area of the ramp. If the contribution for a time period is too large compared to the rest of the time period, it is going to overlap with the neighbor ramps. Thus, the ramps are transparent, so they will not cover their neighbors.
-- As Figure shown, when the user hovers the mouse above a ramp, the time period and the exact amount of contribution will be shown.
-- If you click on a ramp, a github page containing the commits in that period of time will be opened
+- As Figure shown, when the user hovers the mouse a ramp, the time period and the exact amount of contribution will be shown.
+- If you click on a ramp, a GitHub page containing the commits in that period of time will be opened.
+
+#### Total Contribution Bars
+The total amount of code contributed is represented by the **red bars**, and the length of these red bars is proportional to the total contribution of the corresponding author.
+Hovering over the bar shows the exact amount of contribution.
+If the author contributes **too much** compared to other authors in the report, there will be multiple red bars for him.
```

Comment:
Remove ```in the report```. It should be a typo and not required.

If an author contributes way more than the other authors, there will be multiple red bars for him.